recipes/pycoal/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pycoal" %}
+{% set version = "0.2" %}
+{% set sha256 = "64f265d800752a5d001b14c49348e892116915b45dc825b3161dab17bd50546a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy
+    - spectral
+
+test:
+  imports:
+    - pycoal
+
+about:
+  home: https://github.com/capstone-coal/pycoal/
+  license: Apache 2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: 'Python toolkit for characterizing Coal and Open-pit surface mining impacts on American Lands'
+  description: |
+    pycoal is a Python library for processing hyperspectral imagery from remote sensing devices such 
+    as the Airborne Visible/InfraRed Imaging Spectrometer (AVIRIS) and AVIRIS-Next Generation 
+    enabling scientific analysis of Coal and Open-pit surface mining impacts on American Lands (COAL).
+  doc_url: http://pycoal.readthedocs.io/
+  dev_url: https://github.com/capstone-coal/pycoal/
+
+extra:
+  recipe-maintainers:
+    - lewismc
+    - browtayl
+    - heidiaclayton
+    - xiaomei7
```

Comment:
Thanks @jakirkham 

@browtayl, @heidiaclayton, @xiaomei7 this issue relates to https://github.com/capstone-coal/pycoal/issues/34